websites/T/TioAnime/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "author": {
+    "name": "Giingu",
+    "id": "156481332652802048"
+  },
+  "service": "TioAnime",
+  "description": {
+    "es": "TioAnime es un sitio web de streaming donde puedes ver anime con subtítulos en español.",
+    "en": "TioAnime is a streaming website where you can watch anime with Spanish subtitles.",
+    "zh_CN": "TioAnime是一个视频网站，您可以在其中观看带有西班牙语字幕的动漫。"
+  },
+  "url": "tioanime.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/Xp4oPgi.jpg",
+  "thumbnail": "https://i.imgur.com/9TgOPi4.png",
```

Comment:
Try and get a less blurry logo and it doesnt look like the logo on the website...